src/components/stressComment/stressComment.css

```diff
@@ -0,0 +1,23 @@
+.StessCommentContainer {
```

Comment:
Following up on the JS name change.
```suggestion
.StressCommentContainer {
```